test/e2e/framework/metrics_util.go

```diff
@@ -488,13 +488,17 @@ func getSchedulingLatency(c clientset.Interface) (*SchedulingLatency, error) {
 	}
 
 	for _, sample := range samples {
+		if sample.Metric[model.MetricNameLabel] != "scheduler_operation_latency_microseconds" {
+			continue
+		}
+
 		var metric *LatencyMetric = nil
-		switch sample.Metric[model.MetricNameLabel] {
-		case "scheduler_scheduling_algorithm_latency_microseconds":
+		switch sample.Metric["operationType"] {
+		case "scheduling_algorithm":
```

Comment:
Please replace these hard-coded strings with their corresponding const vars you defined above.

Reply:
done